services/cinema_hall.py

```diff
@@ -0,0 +1,22 @@
+from django.db.models import QuerySet
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
```

Comment:
Extend your return type annotation indicating queryset of what model you return. Apply this change here and in the other places